README.md

```diff
@@ -1,73 +1,107 @@
 # dds-rtps
 
-Validation of interoperability of products compliant with [OMG DDS-RTPS standard](https://www.omg.org/spec/DDSI-RTPS/). This is considered one of the core [DDS Specifications](https://www.dds-foundation.org/omg-dds-standard/). See https://www.dds-foundation.org/ for an overview of DDS.
+Validation of interoperability of products compliant with
+[OMG DDS-RTPS standard](https://www.omg.org/spec/DDSI-RTPS/).
+This is considered one of the core
+[DDS Specifications](https://www.dds-foundation.org/omg-dds-standard/).
+See https://www.dds-foundation.org/ for an overview of DDS.
 
-The executables found on the [release tab of this repository](https://github.com/omg-dds/dds-rtps/releases) test discovery, DDS Topic and QoS matching, and interoperability for different QoS settings. The goal is to validate that the implementations perform these functions in compliance with OMG DDS-RTPS standard and can interoperate with each other.
+The executables found on the
+[release tab of this repository](https://github.com/omg-dds/dds-rtps/releases)
+test discovery, DDS Topic and QoS matching, and interoperability for different
+QoS settings. The goal is to validate that the implementations perform
+these functions in compliance with OMG DDS-RTPS standard
+and can interoperate with each other.
 
-# Interoperability Automatic Tests
+## Table of contents
 
-The script `interoperability_report.py` generates automatically
-the verification between two executables of these interoperability tests.
-The tests that the script runs must be defined previosly (for example
-`test_suite.py`).
-Once the script finishes, it generates a report with the result
-of the interoperability tests between both executables.
+* 1\. [Introduction](#introduction)
 
-## Options of interoperability_report
+* 2\. [Run Interoperability Test Manually](#run-interoperability-test-manually)
```

Comment:
we should add a new section before the number 2.
2. Test Suite
Here we should explain how to create a test suite, which are the mandatary things that we need to add to the test suite (dictionary). How that dictionary is created, explaining the checking function, etc...
3. Run Interoperability Test Manually

Reply:
done